packages/next/build/webpack/loaders/next-flight-server-loader.ts

```diff
@@ -73,6 +73,7 @@ async function parseImportsInfo({
             // Since we already include React in the SSR runtime,
             // here we can't create a new module with the ?__rsc_server__ query.
             if (
+              importSource === 'react' ||
               ['react/jsx-runtime', 'react/jsx-dev-runtime'].includes(
                 importSource
               )
```

Comment:
There's a case when the source become `react?__sc_server__` and it will yield error "Cannot resolve ./?__sc_server__" in `.../react/package.json`

I didn't put it in the array because it could be any other library, .e.g if I have some lib like `ohreactyes`